### DISCUSSION TITLE
Hide "/gallery" in url

### DISCUSSION THREAD

**Discussion by wizgod** [bpatrik/pigallery2#867] [General]
Greetings Programs!

I figure this is probably more of an Apache aliasing as root question but how can I hide the "/gallery" path in the URL?

My domain TLD is ".gallery" so the double gallery... well, yeah, probably being a little nit picky, LOL

Thanks,

Wg

**Comment by bpatrik** [comment]
That is not possible. That sting is used on the client app for routing.

-- Sorry for being brief, sent from my phone.
